flixel/FlxSprite.hx

```diff
@@ -59,11 +59,13 @@ class FlxSprite extends FlxObject
 	
 	/**
 	 * Controls whether the object is smoothed when rotated, affects performance.
+	 * @since 4.3.0
 	 */
 	public var smoothing(default, set):Bool = false;
 	
 	/**
 	 * Controls whether the object is smoothed when rotated, affects performance.
+	 * It's better to use `smoothing` property instead, since `antialiasing` is deprecated.
```

Comment:
I meant as an argument for `@:deprecated`, then it will show up in compiler warnings. See here for an example: https://github.com/HaxeFlixel/flixel/blob/dev/flixel/math/FlxRandom.hx#L307